tools/check_crash.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+IOTJS_BIN=$1
+
+trap ':' SEGV
+
+for file in $(ls ./test/crash/*.js)
+do
+  echo "running $file"
+  $IOTJS_BIN $file > /dev/null 2>&1
```

Comment:
We can collect pid of each run
e.g.
```sh
pids="$(sh -c "echo \$\$; exec $IOTJS_BIN $file > /dev/null 2>&1") $pids"
```

then we can determine if the dump file of each process are removed rather than expecting no `/tmp/iotjs.*` exists

Reply:
Good point 👍